sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/ChangeFeedInitialOffsetWriter.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.spark
+
+import org.apache.commons.io.IOUtils
```

Comment:
we don't have explicit dependency on this. Where is this coming from? is this from spark?

Reply:
Yes - my understanding is it is pulled into Spark here: https://github.com/apache/spark/blob/9809a2f1c5187205c81542dbdc84b71db535f6e1/pom.xml#L530-L534

Reply:
Discussed offline - added the external dependency as provided